spring-cloud-commons/src/main/java/org/springframework/cloud/commons/util/IdUtils.java

```diff
@@ -64,7 +65,13 @@ public static String getDefaultInstanceId(PropertyResolver resolver, boolean inc
 	 * @return A unique id that can be used to uniquely identify a service
 	 */
 	public static String getResolvedServiceId(PropertyResolver resolver) {
-		return resolver.resolvePlaceholders(getUnresolvedServiceId());
```

Comment:
Probably worth adding a comment to the code explaining why this logic is here and that we should remove this method in the future in favor of `getUnresolvedServiceIdWithActiveProfile`.  Mark this method as `@Deprecated`

Reply:
Resolved by a51f4b2.

Reply:
Superseded by 91aaf03.